array-map/main.js

```diff
@@ -0,0 +1,21 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+const languages = [
+  'Hypertext Markup Language',
+  'Cascading Style Sheets',
+  'ECMAScript',
+  'PHP Hypertext Preprocessor',
+  'Structured Query Language'
+];
+
+const doubled = numbers.map(number => number * 2);
+console.log('doubled: ', doubled);
+
+const prices = numbers.map(number => `$${number.toFixed(2).toString()}`);
```

Comment:
`.toFixed` already returns a string so there is no need to call `toString`, also because you are concatenating into a string it will be a string no matter what.